src/xsd4/iis.xsd

```diff
@@ -692,9 +692,13 @@
           </xs:restriction>
         </xs:simpleType>
       </xs:attribute>
-      <!-- The Overwrite attribute is not implemented at this time.
-      <xs:attribute name="Overwrite" type="wxs:YesNoTypeUnion" />
-      -->
+      <xs:attribute name="Overwrite" type="wxs:YesNoTypeUnion">
+        <xs:annotation>
+          <xs:documentation>
+            [WiX v5] Allow the certificate to overwrite.
```

Comment:
Still unsupported, no?

Reply:
I wondered about that. There is a test with it, and code was added to the compiler but I didn't see the CA change. I'll take it back out based on your comment.

Reply:
We had no cert unit tests so I added one for https://github.com/wixtoolset/issues/issues/7784.